block_util/src/async_io.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright © 2021 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 AND BSD-3-Clause
+
+use vmm_sys_util::eventfd::EventFd;
+
+#[derive(Debug)]
+pub enum DiskFileError {
```

Comment:
If you're adding a new error please use thiserror/anyhow to avoid us needing to change it later.

Reply:
Sounds good!